src/test/java/com/github/dockerjava/api/model/InfoTest.java

```diff
@@ -0,0 +1,88 @@
+package com.github.dockerjava.api.model;
+
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.dockerjava.api.model.Info.RegistryConfig;
+import com.github.dockerjava.api.model.Info.RegistryConfig.IndexConfig;
+import org.hamcrest.CoreMatchers;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static com.github.dockerjava.core.RemoteApiVersion.VERSION_1_22;
+import static com.github.dockerjava.test.serdes.JSONSamples.testRoundTrip;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.isEmptyString;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+/**
+ * @author Kanstantsin Shautsou
+ */
+public class InfoTest {
+
+    @Test
+    public void serder1Json() throws IOException {
```

Comment:
it will be new serder as it unit test.

ITs should be run against different hosts.